integration/v7/isolated/unset_space_role_command_test.go

```diff
@@ -42,6 +42,7 @@ var _ = Describe("unset-space-role command", func() {
 				Eventually(session).Should(Say("SpaceManager - Invite and manage users, and enable features for a given space"))
 				Eventually(session).Should(Say("SpaceDeveloper - Create and manage apps and services, and see logs and reports"))
 				Eventually(session).Should(Say("SpaceAuditor - View logs, reports, and settings on this space"))
+				Eventually(session).Should(Say("SpaceSupporter -"))
```

Comment:
Could this test be updated to include the help text you added in the next commit?

Reply:
Updated this in the newest commit